From a8a1ac67c56f4a572720927778c9217e7e9e3e2e Mon Sep 17 00:00:00 2001
From: Stanislav Ochotnicky <sochotnicky@redhat.com>
Date: Tue, 18 Feb 2014 21:13:45 +0100
Subject: [PATCH 6/6] Skip automatic requires on java and java-devel

---
 depgenerators/maven.req | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/depgenerators/maven.req b/depgenerators/maven.req
index 8959f3f..650f450 100755
--- a/depgenerators/maven.req
+++ b/depgenerators/maven.req
@@ -166,11 +166,6 @@ class TagBuilder:
                             "%mvn_package {art} <package_name>\n".format(art=skipped)
             raise Exception(skipped_msg)
 
-        reqs = {'java': depmap.get_java_requires(),
-                'java-devel': depmap.get_java_devel_requires()}
-        for name, version in reqs.iteritems():
-            self.__process_jvm_req(name, version)
-
 
 if __name__ == "__main__":
     try:
-- 
1.8.5.3

